airflow-core/src/airflow/dag_processing/processor.py

```diff
@@ -87,6 +85,11 @@ def _parse_file_entrypoint():
     task_runner.SUPERVISOR_COMMS = comms_decoder
     log = structlog.get_logger(logger_name="task")
 
+    # Put bundle root on sys.path if needed. This allows the dag bundle to add
+    # code in util modules to be shared between files within the same bundle.
+    if (bundle_root := os.fspath(msg.bundle_path)) not in sys.path:
+        sys.path.append(bundle_root)
```

Comment:
Should this be append or prepend? What did we do with DAGs folder in 2.x?

Reply:
We do append with the dags folder. I was skeptical about this too but decided to just copy the behaviour for now.

Reply:
Keeping it the same is fine imo.